golos.social/golos.social.cpp

```diff
@@ -113,5 +90,36 @@ void social::unblock(name blocker, name blocking) {
     });
 }
 
+void social::checkblocked(name blocker, name blocking) {
```

Comment:
Why did you choose to do this test in the form of an action? In this place, no change in the social contract data is required and the implementation in the form of reading from the table of another contract will work much faster.
Let's do this method in the form of static functions inline. See: https://github.com/GolosChain/golos-smart/blob/292f300878d976691395af6e6fc3a013d210f1f7/golos.vesting/golos.vesting.hpp#L36-L40

Reply:
@s-medvedev I agree.